Traceback (most recent call last):
  File "/home/jellyfish/MLOpsCamp/mlops-zoomcamp/cohorts/2023/02-experiment-tracking/homework-wandb/preprocess_data.py", line 96, in <module>
    run_data_prep()
  File "/home/jellyfish/anaconda3/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/home/jellyfish/anaconda3/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/home/jellyfish/anaconda3/lib/python3.10/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/jellyfish/anaconda3/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home/jellyfish/MLOpsCamp/mlops-zoomcamp/cohorts/2023/02-experiment-tracking/homework-wandb/preprocess_data.py", line 59, in run_data_prep
    df_train = read_dataframe(
  File "/home/jellyfish/MLOpsCamp/mlops-zoomcamp/cohorts/2023/02-experiment-tracking/homework-wandb/preprocess_data.py", line 17, in read_dataframe
    df = pd.read_parquet(filename)
  File "/home/jellyfish/.local/lib/python3.10/site-packages/pandas/io/parquet.py", line 509, in read_parquet
    return impl.read(
  File "/home/jellyfish/.local/lib/python3.10/site-packages/pandas/io/parquet.py", line 220, in read
    path_or_handle, handles, kwargs["filesystem"] = _get_path_or_handle(
  File "/home/jellyfish/.local/lib/python3.10/site-packages/pandas/io/parquet.py", line 110, in _get_path_or_handle
    handles = get_handle(
  File "/home/jellyfish/.local/lib/python3.10/site-packages/pandas/io/common.py", line 868, in get_handle
    handle = open(handle, ioargs.mode)
FileNotFoundError: [Errno 2] No such file or directory: '/data/green_tripdata_2022-01.parquet'